lib/product.rb

```diff
@@ -0,0 +1,25 @@
+class Product
+  attr_reader :category,
+              :name,
+              :unit_price,
+              :quantity
+
+  def initialize(category, name, unit_price, quantity)
+    @category   = category
+    @name       = name
+    @unit_price = unit_price
+    @quantity   = quantity.to_i
+  end
+
+  def total_price
+    @unit_price * @quantity
+  end
+
+  def is_hoarded?
+    return @quantity > 10
```

Comment:
We don't need the explicit return here

Reply:
Also, products are not hoarded by default, but the hoard method makes them hoarded.